### ISSUE TITLE
Add console.* handling

### ISSUE THREAD

**Issue by OleksiyRudenko** [OleksiyRudenko/jsconsole-bot#3]
As a user I want to see the output of `console.*` invocations from my code snippet

**Comment by OleksiyRudenko** [comment]
Solution:
```
// Create a new sandbox VM for this request
    const vm = new NodeVM( {
        console: 'redirect',
        require: {
            external: ['request']
        }
    });

    vm.on('console.log', (data) => {
        console.log(`VM stdout: ${data}`);
      });
```

[Source](https://stackoverflow.com/questions/50299007/nodejs-vm2-proper-way-to-access-console-when-set-to-redirect)

**Comment by OleksiyRudenko** [comment]
Resolved by 7e7a8a0
